app/assets/stylesheets/pageflow/entries.css.scss

```diff
@@ -108,17 +108,25 @@
       z-index: 0;
     }
   }
-}
 
-.has_mobile_platform .hidden_text_indicator {
-  left: 8%;
-  &:before {
-    background-image: image-url('pageflow/touch_indicator.png');
-  }
+  .hidden_text_indicator {
+    display: none;
+    left: 8%;
 
-  &.invert {
     &:before {
-      background-image: image-url('pageflow/touch_indicator_invert.png');
+      background-image: image-url('pageflow/touch_indicator.png');
     }
+
+    &.invert {
+      &:before {
+        background-image: image-url('pageflow/touch_indicator_invert.png');
+      }
+    }
+  }
+}
+
+.has_mobile_platform .entry {
+  .hidden_text_indicator {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `hidden_text_indicator` should be written in lowercase with hyphens